### ISSUE TITLE
how to specify an option ? (enforce a license for example)

### ISSUE THREAD

**Issue by goulu** [uetchy/create-create-app#36]
is there a way to skip some wizard questions like
```
create("create-myapp", {
  templateRoot,
  license:"MIT",
  author:"me myself"
})
```
Thanks !

(I'm very happy to talk with the creator of the app creators creator ;-) )

**Comment by uetchy** [comment]
You can skip these questions by passing CLI flags to your app!

```
create-myapp --license MIT --author "me myself"
```

**Comment by uetchy** [comment]
Implemented in [v7.3.0](https://github.com/uetchy/create-create-app/releases/tag/v7.3.0)
